packages/menu-bar/test/sub-menu.test.js

```diff
@@ -603,21 +605,21 @@ describe('theme attribute', () => {
     buttons[0].dispatchEvent(new CustomEvent(menuOpenEvent, { bubbles: true, composed: true }));
     await nextRender(subMenu);
 
-    items = subMenu.$.overlay.querySelectorAll('vaadin-menu-bar-item');
+    items = subMenuOverlay.querySelectorAll('vaadin-menu-bar-item');
 
     expect(items[0].getAttribute('theme')).to.equal('sub-item-theme');
     expect(items[1].hasAttribute('theme')).to.be.false;
   });
 });
 
 describe('touch', () => {
-  let menu, buttons, subMenu, items, item;
+  let menu, buttons, subMenu, subMenuOverlay, items, item;
 
   const open = (openTarget) => {
-    const menu = openTarget.parentElement.parentElement.__dataHost;
+    const menu = openTarget.parentElement.parentElement.owner;
```

Comment:
Changed to use `owner` instead of Polymer-specific private API:

https://github.com/vaadin/web-components/blob/c0f9ab4cc9eae036fb8c20a53342e60f21650995/packages/context-menu/src/vaadin-context-menu-mixin.js#L169